client-js/app/elements/elements.html

```diff
@@ -40,3 +43,4 @@
 <link rel="import" href="labrad-server.html">
 <link rel="import" href="menu-button.html">
 <link rel="import" href="selectable-table.html">
+<link rel="import" href="expandable-input.html">
```

Comment:
I would prefer to import things where they are needed, rather than in one global location; I realize we inherited this from the polymer example project, but I think we should change the pattern.